src/AspNet.Security.OAuth.Yahoo/YahooAuthenticationConstants.cs

```diff
@@ -15,8 +15,7 @@ public static class Claims
         {
             public const string FamilyName = "urn:yahoo:familyname";
             public const string GivenName = "urn:yahoo:givenname";
-            public const string ImageUrl = "urn:yahoo:profileimage";
```

Comment:
To keep this as a minor release rather than a breaking-change, could you add these back and add `[Obsolete]` attributes to the old constants explaining  why please?

Reply:
Done!